pkg/proxy/ipvs/proxier.go

```diff
@@ -1008,11 +1008,9 @@ func (proxier *Proxier) syncProxyRules() {
 					serv.Timeout = uint32(svcInfo.StickyMaxAgeSeconds)
 				}
 				if err := proxier.syncService(svcNameString, serv, true); err == nil {
-					// check if service need skip endpoints that not in same host as kube-proxy
-					onlyLocal := svcInfo.SessionAffinityType == v1.ServiceAffinityClientIP && svcInfo.OnlyNodeLocalEndpoints
```

Comment:
This change seems reasonable to me, as you don't need to have serviceAffinity and ExternalTrafficPolicy to only sync local endpoints. From what I see #64146 added this to fix flakiness of an e2e. Let's give e2e some runs and see how it goes.

Reply:
@yastij I would like to know the e2e test results. Have you ever run the tests?

Reply:
@m1093782566 isn't /test all enough?

Reply:
I assume not, I usually use `kubetest --provider=gce --test` to run e2e tests on GCE - this PR is GCE/AWS LB related. Please reference: https://github.com/kubernetes/community/blob/master/contributors/devel/e2e-tests.md

Reply:
@lbernail - nope, not all e2e are played. I did a run for the relevant e2e. will add them later today.

Reply:
Thanks! I assumed e2e tests were run as part of the standard tests